src/lib/utils/utils.ts

```diff
@@ -22,3 +23,34 @@ export const isValidUrl = (url: string): boolean => {
     return false;
   }
 };
+
+export const triggerConfetti = () => {
+  const end = Date.now() + 2 * 1000; // 2 seconds
+  const colors = ['#a786ff', '#fd8bbc', '#eca184', '#f8deb1'];
+
+  const frame = () => {
+    if (Date.now() > end) return;
+
+    confetti({
+      particleCount: 1,
+      angle: 60,
+      spread: 55,
+      startVelocity: 60,
+      origin: { x: 0, y: 0.5 },
+      colors: colors,
+    });
+
+    confetti({
+      particleCount: 1,
+      angle: 120,
+      spread: 55,
+      startVelocity: 60,
+      origin: { x: 1, y: 0.5 },
+      colors: colors,
+    });
```

Comment:
You can simplify like this:
```suggestion
const confettiConfigs = [
  { angle: 60, origin: { x: 0, y: 0.5 } },
  { angle: 120, origin: { x: 1, y: 0.5 } },
];

confettiConfigs.forEach(config => {
  confetti({
    particleCount: 1,
    spread: 55,
    startVelocity: 60,
    colors: colors,
    ...config,
  });
});

```